backend/Infraestructure/IOC/IOCInfraestructure.cs

```diff
@@ -16,14 +18,23 @@ public static class IOCInfraestructure
     {
         public static void AddPersistence(this IServiceCollection services, IConfiguration configuration)
         {
+            #region Context
             services.AddDbContext<ApplicationDbContext>(options =>
             {
                 options.UseSqlite(configuration.GetConnectionString("SQLiteConnection"), b => b.MigrationsAssembly("Infraestructure.IOC"));
             });
+            #endregion
 
-            services.AddIdentity<IdentityUser, IdentityRole>()
+            #region Identity
+            services.AddIdentity<Admin, IdentityRole>()
```

Comment:
Isn't this going to cause a problem? Since you are adding Admin, but we have 3 different entities, Admin, Company and Client.

Reply:
Done!